cpanfile

```diff
@@ -95,6 +95,7 @@ requires 'MooseX::ClassAttribute';
 requires 'MooseX::Getopt';
 requires 'MooseX::Getopt::Dashes';
 requires 'MooseX::Getopt::OptionTypeMap';
+requires 'MooseX::StrictConstructor';
```

Comment:
When you add a new module, make sure you run carton and then commit the changes to cpanfile.snapshot so that we can mimic your installed modules in production.